database/open_source/os-programs.json

```diff
@@ -26,6 +26,13 @@
       "url": "https://owasp.org/events/",
       "category": "open-source",
       "subcategory": "os-programs"
+    },
+     {
+      "name": "HackSquad",
+      "description": "HackSquad allows contributors to contribute code as a squad instead of a single contributor. Engage the community in a friendly competition over the month of October",
```

Comment:
```suggestion
      "description": "HackSquad allows contributors to contribute code as a squad instead of a single contributor. Engage with the community in a friendly competition over the month of October",
```